<launch>
  <arg name="config_path" default="$(find learn_msckf)/config/config_lymow.yaml"/>

  <node name="mono_msckf" pkg="learn_msckf" type="msckf_node" 
      args="-config_file $(arg config_path)" required="true">
    <!-- <remap from="imu0" to="/handsfree/imu"/>
    <remap from="cam0/image_raw" to="/camera1/Image"/> -->
  </node>

  <node name="rvizvisualisation" pkg="rviz" type="rviz" output="log" 
      args="-d $(find learn_msckf)/config/msckf_config.rviz"/>

</launch>
